src/splitInteger.test.js

```diff
@@ -1,21 +1,41 @@
+// src/splitInteger.test.js
+
 'use strict';
 
 const splitInteger = require('./splitInteger');
 
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
+test('should split into equal parts if divisible', () => {
+  const value = 12;
```

Comment:
please don't declare any variables in tests.  Expect statement with values would be enough. e.g expect(splitInteger(8, 4))
    .toEqual([2, 2, 2, 2]);